Assets/Scripts/DataModels/DataUtils.cs

```diff
@@ -103,6 +103,16 @@ public static void CreateSaveFolder(string folderName)
         Directory.CreateDirectory(GetSaveFolderPath(folderName));
     }
 
+    public async static Task<string> ReadTextFileAsync(string fileName)
+    {
+        string text;
+        using (StreamReader reader = File.OpenText(fileName))

```

Comment:
should probably check if file exists first

Reply:
Good shout I'll do that